src/ResourceManager/Compute/Commands.Compute/help/Remove-AzureRmVmss.md

```diff
@@ -27,10 +27,12 @@ You can use the *InstanceId* parameter to remove a specific virtual machine insi
 
 ### Example 1: Remove a VMSS
 ```
-PS C:\> Remove-AzureRmVmss -ResourceGroupName "Group001" -VMScaleSetName "VMScaleSet001"
+PS C:\> $vmss = Get-AzureRmVmss -ResourceGroupName $RGName -VMScaleSetName $vmssName 
+PS C:\> Remove-AzureRmVmssExtension -VirtualMachineScaleSet $vmss -Name $vmssExtensionName
```

Comment:
@cloudmelon shouldn't this example used `Remove-AzureRmVmss`?

Reply:
@cormacpayne you're right... I think my example would be placed at Remove-AzureRMVmssExtension page. You inspired me such a good idea to use this command by passing extension name or extension id, both of them will be working, thanks  :)

Reply:
@cloudmelon would you like me to close this PR, or are you planning on updating this example?

Reply:
Please close it :)  I found the right place now :)